src/pages/coders.js

```diff
@@ -101,7 +101,8 @@ function Coder({ githubUser, avatar, approved, totalContributions }) {
         <CoderDetails>
           <CoderText>
             <CoderName>@{githubUser}</CoderName>
-            {totalContributions} contribuições
+            {totalContributions}{" "}
+            {totalContributions === 1 ? `contribuição` : `contribuições`}
```

Comment:
Could you please use the same react component here?

Reply:
@marcelometal done!